src/System.Private.ServiceModel/tests/Unit/project.json

```diff
@@ -51,6 +50,11 @@
     "xunit.netcore.extensions":  "1.0.0-prerelease-*"
   },
   "frameworks": {
-    "dnxcore50": {}
+    "dnxcore50": {
+      "dependencies": {
+        "System.Net.Sockets": "4.0.10-beta-*",
+        "System.Reflection.Emit.Lightweight": "4.0.0-beta-*",
+      }
```

Comment:
Did you fix the Reflection.Emit.Lightweight dependency everywhere needed -- or is this a left-over from experimentation?